definitions/npm/@ginkgo-bioworks/react-json-schema-form-builder_v1.x.x/flow_v0.30.x-v0.103.x/react-json-schema-form-builder_v1.x.x.js

```diff
@@ -0,0 +1,120 @@
+declare module '@ginkgo-bioworks/react-json-schema-form-builder' {
```

Comment:
Why scoping it to `v0.30.0-v0.103.x` only?

Reply:
whoops, this is a holdover from copying another definition's file structure. We really only tested this with 0.92.1, so I'll ask to support that and newer versions.